src/libs/state.js

```diff
@@ -2,8 +2,25 @@ import { staticStorageSlot } from 'src/libs/browser-storage'
 import * as Utils from 'src/libs/utils'
 
 
+export const authStore = Utils.atom({
+  isSignedIn: undefined,
+  anonymousId: undefined,
+  registrationStatus: undefined,
+  acceptedTos: undefined,
+  user: {},
+  profile: {},
+  fenceStatus: {},
+  cookiesAccepted: undefined,
+  authContext: undefined,
+  oidcConfig: {}
+})
```

Comment:
I am not sure I follow on this one, is this because CBAS-UI has its own way of maintaining auth?

Reply:
Yeah, the auth for Leo apps is handled in a proxy layer. For Azure it's done in the Relay Listener (a separate service within the app), which takes the user's token and checks it with Sam, then forwards the request to the rest of the app if the user is authorized to access it. I'm less familiar with GCP, I think it works similarly but the proxy is in Leo rather than installed as part of the app. 

So, once the user is inside CBAS we know they are authenticated and allowed to be there. And the pod that's running CBAS UI is running with the user's identity, as opposed to being a service that multiple users can be logged in to. It's a different paradigm from a normal web app. As with MixPanel, WM has probably done some thinking about this already, a good related XTTT topic.

Reply:
So this is ultimately only used for a link to the GCP bucket with the execution directory. For now I am leaving this but I filed a ticket for repointing this to the azure blob, and removing the auth store entirely [here](https://broadworkbench.atlassian.net/browse/WX-830)